viewmaster/pdsgrouptable.py

```
@@ -65,42 +114,103 @@ def levels(self):
+        """
+
         return [g for g in self.groups if len(g) > 0]
 
     def iterator_for_all(self):
+        """Return all groups, including those with all members hidden.
+
+        Returns:
+            list: All groups in the table.
+        """
+
         return [g for g in self.groups]
```

Ruff (0.15.5)

[convention] Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)